### PR TITLE
Add Pydantic to `members.csv`

### PR DESCRIPTION
**Pull request by samuelcolvin**
Hi @chadwhitacre, as discussed over email - Pydantic is joining!

### CODE REVIEW

**Review by vladh — CHANGES_REQUESTED** [review]
Hey @samuelcolvin, so nice to have you on board!!! 🎉 

My only comment is that https://pydantic.dev/open-source 404s.

Other than that, LGTM, I'll wait for a second review! :)

**Comment by samuelcolvin** [comment]
🤦 that's criminal incompetency on my end, I've pushed a fix, should be up a in a few minutes.

**Comment by chadwhitacre** [comment]
Dope. Welcome aboard! 💃

**Pull request merged**
